rpc/server.go

```diff
@@ -82,12 +90,26 @@ func NewServer(addr net.Addr, context *Context) *Server {
 	return s
 }
 
-// Register a new method handler. `name` is a qualified name of the form "Service.Name".
-// `handler` is a function that takes an argument of the same type as `reqPrototype`.
-// Both the argument and return value of 'handler' should be a pointer to a protocol
-// message type.
+// Register a new method handler. `name` is a qualified name of the
+// form "Service.Name".  `handler` is a function that takes an
```

Comment:
Old habits die hard. I wish we had a `commentfmt` so no one has to waste time thinking about this.